src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -32,24 +32,29 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.openhtmltopdf.pdfboxout.PdfRendererBuilder;
+import org.testng.*;
 import org.testng.log4testng.Logger;
+import org.testng.xml.XmlSuite;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.testng.xml.XmlSuite'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)